src/lib.rs

```diff
@@ -452,17 +452,41 @@ impl Maildir {
 
     /// Copies a message from the current maildir to the targetted maildir.
     pub fn copy_to(&self, id: &str, target: &Maildir) -> std::io::Result<()> {
-        let src = self.path.join("cur").join(id);
-        let dst = target.path.join("cur").join(id);
-        fs::copy(src, dst)?;
+        let entry = self.find(id).ok_or_else(|| {
```

Comment:
Please squash these changes to `copy_to` and `move_to` into the previous patch.

Reply:
:heavy_check_mark: